### ISSUE TITLE
.jsx instead of .js

### ISSUE THREAD

**Issue by Biuni** [verekia/js-stack-from-scratch#252]
### Type of issue: (feature suggestion, bug?)
Suggestion
### Chapter:
no-chapter

------

Why use `.jsx ` instead of `.js`?  It's not a best practice: https://github.com/facebook/create-react-app/issues/87#issuecomment-234627904

**Comment by verekia** [comment]
Yup, I've switched to `.js` too. It's one of the many things of the tutorial that needs an update :)

**Comment by deyan-a** [comment]
Hey, I can help with the refactor/update, if you're looking for help.
